label_studio/users/api.py

```diff
@@ -45,30 +45,53 @@ def avatar(self, request, pk):
             request.user.save()
             return Response(status=204)
 
-    @swagger_auto_schema(tags=['Users'], operation_summary='Save user details')
+    @swagger_auto_schema(
+        tags=['Users'],
+        operation_summary='Save user details',
+        operation_description='Save details for a specific user, such as their name, contact information, '
+                              'or organization in Label Studio.',
+        request_body=UserSerializer
+    )
     def update(self, request, *args, **kwargs):
         form = UserProfileForm(data=request.data, files=request.FILES, instance=request.user)
         if form.is_valid():
             form.save()
             return Response({'detail': 'user details saved'}, status=200)
 
-    @swagger_auto_schema(auto_schema=None)
+    @swagger_auto_schema(
+        tags=['Users'],
+        operation_summary='List users',
+        operation_description='List the users that exist on the Label Studio server.'
+    )
     def list(self, request, *args, **kwargs):
         return super(UserAPI, self).list(request, *args, **kwargs)
 
-    @swagger_auto_schema(auto_schema=None)
+    @swagger_auto_schema(
+        tags=['Users'],
+        operation_summary='Create new user',
+        operation_description='Create a new user for Label Studio.',
+        request_body=UserSerializer
+    )
     def create(self, request, *args, **kwargs):
         return super(UserAPI, self).create(request, *args, **kwargs)
 
-    @swagger_auto_schema(auto_schema=None)
+    @swagger_auto_schema(
+        tags=['Users'],
+        operation_summary='Get user info',
+        operation_description='Get info about a specific Label Studio user.'
+    )
     def retrieve(self, request, *args, **kwargs):
         return super(UserAPI, self).retrieve(request, *args, **kwargs)
 
     @swagger_auto_schema(auto_schema=None)
```

Comment:
I can't see PATCH method for User. It's better to have swagger description for partial_update() [PATCH] instead of update() [PUT] method, or duplicate it. 
![image](https://user-images.githubusercontent.com/501780/114946481-ea946500-9e53-11eb-9953-e01154b4a613.png)

Reply:
I kept them both replacing "Save" -> "Update" in PATCH method (but still not sure about the purpose of PUT). Can you describe it then @smoreface will edit the descriptions if needed